src/components/helper/AxiosHelper.tsx

```diff
@@ -0,0 +1,62 @@
+import axios from 'axios'
+import { TransactionPayload } from '../../lib/types'
+import { baseURL } from '../../config'
+
+const apiEp = baseURL + '/transactions'
+
+//get Authorization header
+const getUserToken = () => sessionStorage.getItem('token')
+
+export const getAllTransactions = async () => {
+  try {
+    const user = getUserToken()
+    if (!user) {
```

Comment:
Can't this check be done in getUserToken function do it doesn't have to be repeated multiple times?